package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "2.11.0",
+  "version": "2.12.0",
```

Comment:
This didn't get updated in the package-lock.json. Dunno if you need to do `npm i` or something to update it.
I think we forgot to update the lock file previously and the world hasn't ended... wait, COVID, riots, umm...

Reply:
We found the root cause of 2020's problems! 

Will look into this 👍

Reply:
Updated in next commit!